interview-questions.md

```diff
@@ -7,44 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: The foreign key is used to reference the primary key of a table. To correct this mistake the developer must go into the Student model and add the foreign key. In this case the foreign key would be Cohort_id.
 
-  Researched answer:
+  Researched answer: A foreign key is a column or group of columns in a relational database table that acts as a cross-reference between tables because it references the primary key of another table. To fix the issue stated above, we must first go back to the Cohort model to obtain the foreign key from one of the columns and apply it to the Students model. The foreign key would be Cohort_id.
```

Comment:
["A foreign key is a column or group of columns in a relational database table that provides a link between data in two tables. It acts as a cross-reference between tables because it references the primary key of another table, thereby establishing a link between them." ](https://www.techopedia.com/definition/7272/foreign-key#:~:text=No%20Action-,What%20Does%20Foreign%20Key%20Mean%3F,establishing%20a%20link%20between%20them.)

I think your answer is solid but you are shorting yourself by not writing it out in your own words. Anyone can copy and paste.